packages/neural-amp-modeler-lv2/PKGBUILD

```
@@ -0,0 +1,69 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: LekKit <github.com/LekKit>
+# Contributor: Christopher Arndt <aur -at -chrisarndt -dot- de>
+
+declare -g -A submodules=(
+  ["lv2"]="e9d94328743d630e27a9d322015437fd9080695d"
+  ["NeuralAudio"]="84ef71b3189685dbab7a4757f19907dcb9e1d803"
+  ["NeuralAmpModelerCore"]="e181f61efb8d05d34add45b5eecb3893ff21177c"
+  ["RTNeural"]="5909c44909cd6100367f62cd04b348de85d57dbf"
+  ["RTNeural-NAM"]="720d83fc30031eac0a639c60df3d9438930c40d9"
+)
+
+pkgname=neural-amp-modeler-lv2
+pkgver=0.1.6
+pkgrel=1
+pkgdesc='Neural Amp Modeler (NAM) LV2 plugin'
+arch=(aarch64 x86_64)
```

Hm yea, aarch64 seems to fail at some RTNeural code, couldn't find any hints so far but let's remove the arch for now and open an issue upstream
```suggestion
arch=(x86_64)
```

---

Remove `aarch64`.